soccer/gameplay/robocup-py.cpp

```diff
@@ -144,19 +170,18 @@ void OurRobot_move_to_end_vel(OurRobot* self, Geometry2d::Point* endPos,
 }
 
 void OurRobot_move_to(OurRobot* self, Geometry2d::Point* to) {
-    if (to == nullptr) throw NullArgumentException("to");
+    if (to == nullptr) {
+        throw NullArgumentException("to");
+    }
     self->move(*to);
 }
 
-void OurRobot_settle(OurRobot* self) { self->settle(std::nullopt); }
-
-void OurRobot_settle_w_bounce(OurRobot* self, Geometry2d::Point* bounceTarget) {
```

Comment:
This removes the one touch functionality in the path planned layer. We'd have to revert back to the previous version, which I think just used move

Reply:
Reverted.